2017-11-23 17:32:15,480 - log.py[line:34] - INFO: 测试开始时间：2017-11-23_17_32_15
2017-11-23 17:32:15,490 - server.py[line:19] - INFO: start /b taskkill /F /IM node.exe
2017-11-23 17:32:15,500 - server.py[line:21] - INFO: 启动appium服务
2017-11-23 17:32:15,500 - server.py[line:24] - INFO: start /b appium -a 127.0.0.1 -p 4723 --bootstrap-port 9517 --session-override --command-timeout 600
2017-11-23 17:32:15,520 - server.py[line:28] - INFO: http://localhost:4723/wd/hub/
2017-11-23 17:32:20,529 - connectionpool.py[line:208] - DEBUG: Starting new HTTP connection (1): 127.0.0.1
2017-11-23 17:32:22,542 - connectionpool.py[line:396] - DEBUG: http://127.0.0.1:8888 "GET http://localhost:4723/wd/hub/ HTTP/1.1" 502 None
2017-11-23 17:32:22,542 - server.py[line:37] - ERROR: 启动appium服务失败 exceptions must be old-style classes or derived from BaseException, not NoneType
2017-11-23 17:32:22,542 - log.py[line:88] - INFO:   start 初始化用例脚本.
